test/framework/deployment_helpers.go

```
@@ -383,33 +385,72 @@ func DeployUnevictablePod(ctx context.Context, input DeployUnevictablePodInput)
 		Deployment: workloadDeployment,
 	})
 
-	budget := &v1beta1.PodDisruptionBudget{
-		TypeMeta: metav1.TypeMeta{
-			Kind:       "PodDisruptionBudget",
-			APIVersion: "policy/v1beta1",
-		},
-		ObjectMeta: metav1.ObjectMeta{
-			Name:      input.DeploymentName,
-			Namespace: input.Namespace,
-		},
-		Spec: v1beta1.PodDisruptionBudgetSpec{
-			Selector: &metav1.LabelSelector{
-				MatchLabels: map[string]string{
-					"app": "nonstop",
+	// TODO(oscr): Remove when Kubernetes 1.20 support is dropped.
+	serverVersion, err := workloadClient.ServerVersion()
+	if err == nil {
```

Let's fail if an error occurs here (instead of skipping), ~
````
Expect(err).ToNot(HaveOccured)
````